lib/Workflow/Condition/CheckReturn.pm

```diff
@@ -63,11 +62,14 @@ sub evaluate {
     } elsif ( $arg =~ /^[a-zA-Z0-9_]+$/ ) {    # alpha-numeric, plus '_'
         $argval = $wf->context->param($arg);
     } else {
+        local $@;
+        ###TODO any exceptions are ignored???
         $argval = eval $arg;
```

Comment:
I'm thinking that any errors during the evaluation here should be propagated as a workflow error (not as a validation error), because there's a problem executing the workflow...